.github/workflows/llm_bench-python.yml

```diff
@@ -151,9 +151,11 @@ jobs:
           rm -rf ./ov_models/internvl2-1B
       - name: WWB Tests
         run: |
-          pip install git+https://github.com/huggingface/optimum-intel.git
-          GIT_CLONE_PROTECTION_ACTIVE=false PIP_PRE=1 PIP_EXTRA_INDEX_URL=https://storage.openvinotoolkit.org/simple/wheels/nightly pip install ${{ env.WWB_PATH }}
+          python -m pip install ${{ env.WWB_PATH }}
           python -m pytest -v ${{ env.WWB_PATH }}/tests
+        env:
+          PIP_PRE: 1
+          PIP_EXTRA_INDEX_URL: "https://storage.openvinotoolkit.org/simple/wheels/pre-release https://storage.openvinotoolkit.org/simple/wheels/nightly"
   stateful:
```

Comment:
@eaidova is not it this job obsolete? Looks like both `build` and `stateful` jobs do the same things as _stateful_ mode is default.

Should I drop the second job?

Reply:
yes, I think stateful can be dropped